### DISCUSSION TITLE
Geo Coordinates Field

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#3126] [Ideas] [answered]
I have been an ELK stack user for a long time.  I love the coordinate map so I can see where traffic is coming from.

I see that the SO stack is doing the geo coordinates, but are not placing them in a geo_point field type as shown here

![image](https://user-images.githubusercontent.com/78620004/109086720-06378680-76c1-11eb-84ec-cd4541cf7af7.png)

I am not seeing it specified in the various index templates.  Is this something that would be beneficial across the community?

**Comment by weslambert** [comment, marked as answer]
We used to do this, but it looks like the template has not been updated for the new field naming convention.  I've put in an issue to add this back.

https://github.com/Security-Onion-Solutions/securityonion/issues/3152

**Reply by oneCrazyAdmin** [reply]
Thank you! I am a very visual person and there is something soothing about seeing where your attackers are coming from...

**Reply by DebianGuru** [reply]
Any update on this?  I love the maps, in Kibana, but they require this format.  If it's not in the immediate plans to add back to SO, how hard would it be to generate those key  pairs?  I'm guessing something in Logstash?  I'm a real novice there and have no desire to break my box.

**Reply by oneCrazyAdmin** [reply]
It looks like they are going to add it into 2.3.60.  If they stay on their release schedule, should be in under a month.

**Reply by DebianGuru** [reply]
That's awesome!  These guys have it together. The big companies sure could learn a lesson from them.  Ask Google or Microsoft for something and it may take years or never to be done (even things that should be simple).
